terraform/app/modules/paas/aks_database.tf

```
@@ -17,4 +17,5 @@ module "postgres" {
   azure_enable_high_availability = var.postgres_enable_high_availability
   server_version                 = 14
   azure_maintenance_window       = var.azure_maintenance_window
+  server_docker_image            =  "postgis/postgis:14-3.4"
```

Lets hope Terraform FMT check was failing because the extra space:
```suggestion
  server_docker_image            = "postgis/postgis:14-3.4"
```